<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ESPImagePropDialog</class>
 <widget class="QDialog" name="ESPImagePropDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ESP Image Properties</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox1" >
     <property name="title" >
      <string>ESP Image Bounding Box</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="show_esp_bbox_checkbox" >
        <property name="text" >
         <string>Show Bounding Box</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="highlight_atoms_in_bbox_checkbox" >
        <property name="text" >
         <string>Highlight Atoms Inside Bounding Box</string>
        </property>
        <property name="checked" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="select_atoms_btn" >
        <property name="text" >
         <string>Select Atoms Inside Volume</string>
        </property>
        <property name="autoDefault" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>92</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_btn" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string>Alt+O</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
       <property name="default" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>101</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" >
    <widget class="QGroupBox" name="groupBox5" >
     <property name="title" >
      <string>ESP Results Image</string>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="calculate_esp_btn" >
          <property name="text" >
           <string>Calculate ESP</string>
          </property>
          <property name="autoDefault" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_6" >
          <property name="text" >
           <string>xaxisOrient :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="xaxis_spinbox" >
          <property name="maximum" >
           <number>1</number>
          </property>
          <property name="minimum" >
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QToolButton" name="load_btn" >
          <property name="text" >
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="clear_btn" >
          <property name="text" >
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1_6_2" >
          <property name="text" >
           <string>yaxisOrient :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="yaxis_spinbox" >
          <property name="maximum" >
           <number>1</number>
          </property>
          <property name="minimum" >
           <number>-1</number>
          </property>
          <property name="value" >
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLineEdit" name="png_fname_linedit" >
          <property name="sizePolicy" >
           <sizepolicy>
            <hsizetype>7</hsizetype>
            <vsizetype>7</vsizetype>
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="choose_file_btn" >
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset>:icons/ESPImagePropDialog_image1</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QToolButton" name="rotate_ccw_btn" >
          <property name="toolTip" >
           <string>Rotate  90 degrees counter clock-wisely.</string>
          </property>
          <property name="text" >
           <string>+90</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="rotate_cw_btn" >
          <property name="toolTip" >
           <string>Rotate  90 degrees clock-wisely.</string>
          </property>
          <property name="text" >
           <string>-90</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="flip_btn" >
          <property name="toolTip" >
           <string>Flip the image vertically (top to bottom).</string>
          </property>
          <property name="text" >
           <string>Flip</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mirror_btn" >
          <property name="toolTip" >
           <string>Flip image horizontally (left to right).</string>
          </property>
          <property name="text" >
           <string>Mirror</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="textLabel1_4" >
         <property name="text" >
          <string>Name :</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="colorTextLabel_3" >
         <property name="text" >
          <string>Fill Color :</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="colorTextLabel_4" >
         <property name="text" >
          <string>Border Color :</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel1" >
         <property name="text" >
          <string>Width :</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel1_3" >
         <property name="text" >
          <string>Height Offset :</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel1_5" >
         <property name="text" >
          <string>Edge Offset :</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel1_2" >
         <property name="text" >
          <string>Resolution :</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLabel1_2_2" >
         <property name="text" >
          <string>Opacity:</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLineEdit" name="name_linedit" >
         <property name="text" >
          <string/>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <layout class="QVBoxLayout" >
           <property name="margin" >
            <number>0</number>
           </property>
           <property name="spacing" >
            <number>6</number>
           </property>
           <item>
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <widget class="QLabel" name="fill_color_pixmap" >
               <property name="sizePolicy" >
                <sizepolicy>
                 <hsizetype>5</hsizetype>
                 <vsizetype>5</vsizetype>
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>40</width>
                 <height>0</height>
                </size>
               </property>
               <property name="frameShape" >
                <enum>QFrame::Box</enum>
               </property>
               <property name="frameShadow" >
                <enum>QFrame::Plain</enum>
               </property>
               <property name="scaledContents" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="choose_fill_color_btn" >
               <property name="enabled" >
                <bool>true</bool>
               </property>
               <property name="toolTip" >
                <string>Change color</string>
               </property>
               <property name="text" >
                <string>Choose...</string>
               </property>
               <property name="autoDefault" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" >
                <size>
                 <width>16</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <widget class="QLabel" name="border_color_pixmap" >
               <property name="sizePolicy" >
                <sizepolicy>
                 <hsizetype>5</hsizetype>
                 <vsizetype>5</vsizetype>
                 <horstretch>1</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize" >
                <size>
                 <width>40</width>
                 <height>0</height>
                </size>
               </property>
               <property name="frameShape" >
                <enum>QFrame::Box</enum>
               </property>
               <property name="frameShadow" >
                <enum>QFrame::Plain</enum>
               </property>
               <property name="scaledContents" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="choose_border_color_btn" >
               <property name="enabled" >
                <bool>true</bool>
               </property>
               <property name="toolTip" >
                <string>Change color</string>
               </property>
               <property name="text" >
                <string>Choose...</string>
               </property>
               <property name="autoDefault" >
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" >
                <size>
                 <width>16</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <widget class="QLineEdit" name="width_linedit" >
               <property name="maximumSize" >
                <size>
                 <width>80</width>
                 <height>32767</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="textLabel2" >
               <property name="text" >
                <string>Angstroms</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" >
                <size>
                 <width>16</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <widget class="QLineEdit" name="image_offset_linedit" >
               <property name="maximumSize" >
                <size>
                 <width>80</width>
                 <height>32767</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="textLabel2_2" >
               <property name="text" >
                <string>Angstroms</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" >
                <size>
                 <width>16</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <widget class="QLineEdit" name="edge_offset_linedit" >
               <property name="maximumSize" >
                <size>
                 <width>80</width>
                 <height>32767</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="textLabel2_3" >
               <property name="text" >
                <string>Angstroms</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" >
                <size>
                 <width>16</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <widget class="QSpinBox" name="resolution_spinbox" >
               <property name="maximum" >
                <number>512</number>
               </property>
               <property name="minimum" >
                <number>1</number>
               </property>
               <property name="value" >
                <number>20</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" >
                <size>
                 <width>95</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" >
             <property name="margin" >
              <number>0</number>
             </property>
             <property name="spacing" >
              <number>6</number>
             </property>
             <item>
              <widget class="QLineEdit" name="opacity_linedit" >
               <property name="maximumSize" >
                <size>
                 <width>40</width>
                 <height>32767</height>
                </size>
               </property>
               <property name="maxLength" >
                <number>5</number>
               </property>
               <property name="readOnly" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="opacity_slider" >
               <property name="maximum" >
                <number>100</number>
               </property>
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>16</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>name_linedit</tabstop>
  <tabstop>choose_fill_color_btn</tabstop>
  <tabstop>choose_border_color_btn</tabstop>
  <tabstop>resolution_spinbox</tabstop>
  <tabstop>show_esp_bbox_checkbox</tabstop>
  <tabstop>highlight_atoms_in_bbox_checkbox</tabstop>
  <tabstop>select_atoms_btn</tabstop>
  <tabstop>ok_btn</tabstop>
  <tabstop>cancel_btn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ok_btn</sender>
   <signal>clicked()</signal>
   <receiver>ESPImagePropDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_btn</sender>
   <signal>clicked()</signal>
   <receiver>ESPImagePropDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
